docs/effects/hangman.rst

```diff
@@ -108,16 +108,15 @@ effect signature:
          Guess : (x : Char) ->
                  sig MysteryRules Bool
                      (Mystery (Running (S g) (S w)))
-                     (\inword =>
-                            Mystery (case inword of
-                                 True => (Running (S g) w)
-                                 False => (Running g (S w))))
+                     (\inword => if inword
+                                 then Mystery (Running (S g) w)
+                                 else Mystery (Running g (S w)))
          Won  : sig MysteryRules () (Mystery (Running g 0))
                                     (Mystery NotRunning)
          Lost : sig MysteryRules () (Mystery (Running 0 g))
                                     (Mystery NotRunning)
          NewWord : (w : String) -> 
-                   sig MysteryRules () h (Mystery (Running 6 (length (letters w))))
+                   sig MysteryRules () (Mystery NotRunning) (Mystery (Running 6 (length (letters w))))
```

Comment:
Did you mean to make this change? Surely `NewWord` should work for both `NotRunning` and `Running`? (though I guess it isn't important for this particular version of the code).